src/app/modules/main/stickers/models/sticker_pack_list.nim

```diff
@@ -0,0 +1,141 @@
+import NimQml, Tables, sequtils, sugar
+import ./sticker_list
+import status/utils as status_utils
+import status/types/[sticker]
+import ../../../../../app_service/service/stickers/dto/stickers as stickers_dto
```

Comment:
@jrainville same as above, models should work with view items, don't use dto, a purpose of dto is not this one.